joomla/joomla-cms
# [4.0] Remove Settings text from legend

### Summary of Changes
Remove `Settings` in legends to be consistent with the other configuration pages.


### Testing Instructions
Go to Global Configurations.
See legends with the word `Settings`.

Go to the other configuration pages.
See legends without the word `Settings`.
